DataProcessing/datax-host/pom.xml

```diff
@@ -118,13 +118,13 @@ SOFTWARE
         <dependency>
             <groupId>com.microsoft.datax</groupId>
             <artifactId>datax-core_2.4_2.11</artifactId>
-            <version>1.2.3-SNAPSHOT</version>
+            <version>1.2.4-SNAPSHOT</version>
```

Comment:
Please rev the version of all the datax packages to "1.2.4-SNAPSHOT" including datax-keyvault and datax-udf-samples. We normally keep the same versions for all the packages

Reply:
I updated all of the versions to 1.2.4-SNAPSHOT